plugins/voxel_shape_generator.js

```diff
@@ -42,9 +42,10 @@ function openMappingsBox(){
                 type: 'select',
                 options: {
                     momap: 'Mojang Mappings',
-                    mcp: 'MCP'
+                    mcp: 'MCP',
+                    yarn: 'Yarn'
                 },
-                default: 'MCP'
+                default: 'Mojang Mappings'
```

Comment:
This should point to the key, not the value. So "momap". You can also omit it entirely, that way the first option will be the default.